### DISCUSSION TITLE
Can you please use the video card to do some experiment on other displays that you are comfortable with?

### DISCUSSION THREAD

**Discussion by cjakfskvnad** [aiaf/Stillcolor#9] [General]
You really did a great job. I am deeply grateful. But I just want to check if it is the true reason that we get eye strain.Can you please use the video card to do some experiment on other displays that you are comfortable with? Just like the one you put on youtube. That will be great. Thank you for your effort again!

**Comment by aiaf** [comment]
Hi! So there are a couple variables when it comes to external displays and temporal dithering:

- Whether the GPU outputs a temporally dithered image to the external display
- Whether the display applies its own temporal dithering (aka FRC) on the received image to reach its marketed bit depth

Stillcolor takes care of the first point by disabling temporal dithering at the GPU level, and my capture card experiments verify that. 
It means that the external display gets a clean non-temporally dithered signal. The capture card only tells us what the GPU/DCP sent, but not what the display is doing with that signal. Which takes us to point number 2.

There are 2 types of displays when it comes to bit depth. Let's say a manufacturer claims a panel is 10 bits of color, it could either mean true 10-bit (rare, expensive), or 8-bit + FRC (common, affordable). If it's the latter it means when the panel gets the 10-bit signal from your Mac, the TCON (Timing Controller) on the panel will apply a form of FRC i.e. temporal dithering to reach 10 bits of perceived color. A website like https://www.displayspecifications.com/ will tell you whether your display is true 8 or 10-bit or 6/8-bit + FRC, but Display Specifications is not always reliable- it's best to contact the manufacturer and ask them directly.

Ideally you want to get a display without FRC. Get a true 8-bit or 10-bit display and use Stillcolor to get rid of temporal dithering at both ends of the signal.

Fortunately, even if your display is not true 10-bit (i.e. 8 bit + FRC), there's still a way to prevent the display from applying FRC and that is by forcing your Mac to output at a lower bit depth. And the way to do that (right now, on M-series Macs) is by using a low bandwidth HDMI cable. I've had success by using an old *High Speed HDMI Cable with Ethernet* which cannot carry 4k+ 10-bit @ 60Hz, so when the DisplayPort driver performs bandwidth negotiation it asks for a lower bit depth to be able to successfully send the image at the set resolution and refresh rate.

Hope this helps.

**Reply by madmozg** [reply]
I think you should add this explanation to wiki page in this repo :)

**Reply by aiaf** [reply]
@madmozg Absolutely! I think this and a bunch of other findings from the LED Strain thread should eventually make it here.
